incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
```

Comment:
@prydonius what repo would we put it in?  I kinda like it here to allow for easy updates?